align_system/configs/adm_component/regression/comparative.yaml

```diff
@@ -0,0 +1,7 @@
+_target_: align_system.algorithms.comparative_regression_adm_component.ComparativeRegressionADMComponent
+
+structured_inference_engine: ${ref:adm.structured_inference_engine}
+num_samples: 2
```

Comment:
If greedy sampling is the default we should make the default num_samples 1

Reply:
Good catch Jadie (I'll make that change), and apologies I should have mentioned making the default the greedy sampler as I know that's something we discussed but didn't commit to and probably has some impact on your latest experiment setup.